src/app/guards/TenantGuard/index.tsx

```diff
@@ -21,9 +23,14 @@ function TenantGuard({ children }: BaseComponentProps) {
 
     const hasAnyAccess = useUserInfoSummaryStore((state) => state.hasAnyAccess);
     const mutate = useUserInfoSummaryStore((state) => state.mutate);
+    const usedSSO = useUserStore((state) => state.userDetails?.usedSSO);
 
     const showOnboarding = !hasAnyAccess || showBeta;
     if (showOnboarding) {
+        if (usedSSO && !showBeta) {
+            return <SsoUserMessage />;
+        }
```

Comment:
Need to make sure if someone logging in the first time with SSO they can still create a tenant